### ISSUE TITLE
Error : One of your files was not accepted by the system. Please contact Support for assistance.

### ISSUE THREAD

**Issue by mok33** [jpickup/GarminTools#5]
Hi,
I tried to upload the .FIT files created by your tool I get the following error : 
`One of your files was not accepted by the system. Please contact Support for assistance.`
I also tried the ExampleWorkoutSchedule.xls you provided. And I get the same error in https://connect.garmin.com/modern/import-data.

**Comment by jpickup** [comment]
I've only ever tried transferring the FIT files directly to the watch in the NEWFILES folder. I've not tried sending them to Garmin Connect. Can you try that or are you looking to further edit the files in Connect?

**Comment by mok33** [comment]
Oh I see ! my bad then. I am gonna transfer them via USB when I get back home. thank you ! Really a big fan of your tool btw.

**Comment by jpickup** [comment]
Great, I'm glad you like it. I wrote everything for the Fenix 3 and so don't know for sure if it works for other models. In theory it should as the F3 is now quite old and so all modern Garmin watches should be more capable.

The F3 did have a limit on the number of scheduled workouts, so for example a long programme with > ~100 scheduled workouts was truncated. It just meant I had to re-upload every couple of months to get the next batch on the watch. This is also the reason why scheduled workouts in the past are not sent to the watch.
